template/python3-flask-debian/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM openfaas/of-watchdog:0.7.2 as watchdog
+FROM python:3-slim-buster
```

Comment:
I suspect we should pin a specific Python 3 version, if not, folks will run into issues with libraries.

Reply:
(Specific major version)